### PR TITLE
Tasks 21, 22, 23, 24 and 11

### PR DESCRIPTION
**Pull request by awmpietro**

### CODE REVIEW

File: _hooknode/Controller/HookNode.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Controller;
+
```
Comment (rfornea) [inline _hooknode/Controller/HookNode.php:3]: @awmpietro   All the code looks good (although I'm not really familiar with databases) but I think some of these files are in the wrong repository.  

The files that are involved in the following:
-storing or creating a DB of hook nodes
-adding a hook node to the db
-selecting a hood node
-modifying a hook node score
-modifying a hook node status 

...All would need to be in the brokernode repository.  

Basically, the brokernode needs to have a DB of hook nodes and the methods to interact with that DB.  

The hooknode needs to have a DB of broker node and the methods to interact with that DB.

**Review by rfornea — CHANGES_REQUESTED** [review]
Left a comment with some changes that need to be made.  The code itself looks good, it's just the wrong repo for some of the files.

File: _hooknode/Controller/HookNode.php (hunk 1 above)
Comment (awmpietro) [reply _hooknode/Controller/HookNode.php:3]: Ok, Thanks @rfornea. I'll change it

Comment (awmpietro) [reply _hooknode/Controller/HookNode.php:3]: Submitted to the other repo. Review please, i will cancel this PR
